style.css

```diff
@@ -0,0 +1,77 @@
+:root {
+  /* colors */
+
+  --primary-color: #172b4d;
+  --secondary-color: #7f8cff;
+
+  /* fonts sizes */
+
+  --fw-400: 400;
+  --fw-500: 500;
+  --fw-600: 600;
+  --fw-700: 700;
+  --fw-800: 800;
+  --fw-900: 900;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+  text-decoration: none;
+}
```

Comment:
kindly the body must have a background image according to the [Figma template designe](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m1_setup/headline-mobile.png)